packages/web3-net/src/index.ts

```diff
@@ -0,0 +1,18 @@
+import { Web3Context } from 'web3-core';
+import { EthExecutionAPI } from 'web3-common';
+
+import * as rpcMethodsWrappers from './rpc_method_wrappers';
+
+export default class Web3Net extends Web3Context<EthExecutionAPI> {
```

Comment:
We really don't need `EthExecutionAPI` here rather just `net` specific API. if you define it in package locally and use it, that will probably be fine.